sdk/iot/hub/samples/README.md

```diff
@@ -17,9 +17,12 @@ This document explains samples for the Azure SDK for Embedded C IoT Hub Client a
 
 Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
 
-## Samples for Azure IoT Hub Client APIs
+## Prerequisites
 
-This document describes how to use samples and what is done in each sample.
+- To generate the device certificate, the provided script uses [OpenSSL](https://www.openssl.org/). Please
```

Comment:
I think this also comes from VCPKG @CIPop

Reply:
[I think that might be curl](https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md#vcpkg)

Reply:
Yes: all prereqs come from vcpkg.